_posts/2024-01-19-yannick-horrix-opentofu.md

```diff
@@ -0,0 +1,160 @@
+As such, both currently offer the same large set of features making them well suited tools for IaC workloads.
+
+Some small features were added with the release of OpenTofu 1.6.0 however, including:
+* An advanced testing feature for improved stability in configurations and modules
+* Enhanced S3 state backend with new authentication methods
+* A new provider and module registry, allowing a simpler method of publishing via pull request
+
+The upcoming version 1.7 aims to introduce more community requested features not available in Terraform, including:
+* Client-side state encryption for heightened security in regulated environments
+* Parameterizable backends, providers, and modules to enable more readable, DRY code
+* Third-party extensibility, with a plugin system for new state backends
+
+On Terraform's side, version 1.6 implemented performance improvements to the 'terraform test' command, as well as changes to the S3 backend.
+
+For now, both tools are functionally equivalent and compatible. 
+Time will tell whether this will remain the case as development continues on both sides.
+## Configuration and Syntax
+The core difference between OpenTofu and Terraform lies in its syntax. 
+OpenTofu uses a declarative syntax, where users can define their desired end state without having to worry about the steps required to get there. 
+Terraform, on the other hand, employs a procedural approach. 
+This means users have to define the specific steps, leading to higher control over the execution at the cost of higher complexity.
+
+In terms of configuration, OpenTofu focuses on simplicity in its configuration language, prioritizing readability and ease of use. 
+Terraform configuration, on the other hand, can be more verbose and complex to maintain.
+## Licensing and Pricing
+Both Terraform and OpenTofu are open-source tools, providing free usage.
+
+As said, OpenTofu uses the Mozilla Public License (MPL), which allows developers to freely use, modify, and distribute software for both commercial and non-commercial use.
+
+Terraform has now shifted to the Business Source License (BUSL). This also permits free use of source code, including commercial use, except when it’s used to provide an offering that competes with HashiCorp.
+An example of this could be running Terraform in a hosted way in CI/CD and offering this as a production service.
+
+So while, Terraform remains free to use in your projects, there can be an issue when incorporating Terraform in a service offering.
+
+In terms of paid offerings, Terraform does offer multiple options with Terraform Cloud, a platform managing Terraform code, availaible as a hosted service. 
+Additionally, organizations with advanced security and compliance needs can purchase Terraform Enterprise. 
+This is a self-hosted distribution of Terraform Cloud which offers a private instance that includes the advanced features available in Terraform Cloud.
```

Comment:
Add pricing details, referencing a date, so people get an idea of the impact of using TF Cloud on your cloud bill (it's significant since the recent resource based pricing)